src/index.js

```diff
@@ -10,10 +10,23 @@ const addEventsUrl = baseUrl + '/addEvents';
 const uploadLogsUrl = baseUrl + '/api/uploadLogs';
 
 let logGroupOptions = {};
-if (process.env['LOG_GROUP_OPTIONS']) logGroupOptions = JSON.parse(process.env['LOG_GROUP_OPTIONS']);
-let defaultParserName = process.env['PARSER_NAME'];
+let userLogFile;
+let userParserName;
+let userServerHost;
+if (process.env['LOG_GROUP_OPTIONS']) {
+    logGroupOptions = JSON.parse(process.env['LOG_GROUP_OPTIONS']);
+    let functionLogGroupName = process.env['AWS_LAMBDA_LOG_GROUP_NAME'];
+    if (logGroupOptions[functionLogGroupName]) {
+        userLogFile = logGroupOptions[functionLogGroupName]['logfile'];
+        userServerHost = logGroupOptions[functionLogGroupName]['serverHost'];
+        userParserName = logGroupOptions[functionLogGroupName]['parser'];
+    }
+}
+
+let defaultParserName = (userParserName || process.env['PARSER_NAME']);
 if (!defaultParserName || !defaultParserName.length) defaultParserName = 'cloudWatchLogs';
-let defaultServerHost = process.env['SERVER_HOST'];
+
+let defaultServerHost = (userServerHost || process.env['SERVER_HOST']);
```

Comment:
Maybe this should be the other way around (i.e. if the SERVER_HOST env var is explicitly set, that should take precedence). Same with defaultParserName a couple of lines above